4.x/ocp4_vars.yml

```diff
@@ -25,3 +25,10 @@ student_workloads: []
 # discouraged to do so.
 # default_workloads: [ocp4-workload-enable-lets-encrypt-certificates]
 
+clientvm_instance_type: "t2.medium"
+clientvm_instance_count: 1
+master_instance_type: "m5.xlarge"
+master_instance_count: 3
```

Comment:
according to OpenShift docs, 3 masters (xlarge+) and 2 compute (large+) is a minimum requirement.

Reply:
The recommendation may be for a production cluster.

I was hoping we could run with less masters for dev clusters.
In past we would run 1 Master and 1 (or 2) worker nodes.


How about we try with 1 master, see how it behaves.  If we see issues we bump to more masters?

Reply:
I tried running with one master. But my installation did not proceed after 85%. the openshift installer then timed out. I can try again.